ide/models/scriptfile.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+
+from django.utils.timezone import now
+from django.db import models
+
+from ide.models.s3file import S3File
+
+
+class ScriptFile(S3File):
```

Comment:
In fact, the comment is wrong. It should say "ScriptFiles add support to S3Files for last-modified timestamps and code folding". Which makes me think that TextFile is a reasonable alternative name